examples/annotated-skaffold.yaml

```diff
@@ -110,15 +110,16 @@ deploy:
     # helm releases to deploy.
     # releases:
     # - name: skaffold-helm
-    #   chartPath: skaffold-helm
-    #   valuesFilePath: helm-skaffold-values.yaml
-    #   values:
-    #     image: skaffold-helm
-    #   namespace: skaffold
-    #   version: ""
-    #
-    #   # setValues get appended to the helm deploy with --set.
-    #   setValues:
+    #  chartPath: skaffold-helm
+    #  valuesFilePath:
```

Comment:
Should the new field perhaps be plural? `valuesFilePaths`
Perhaps double plural is a bit strange; but we do have multiple paths, each for a values.yaml file.

Reply:
+1 on plural - maybe simply `valuesFiles`?